src/css_provider.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013-2015, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use Error;
+use CssProvider;
+use ffi;
+use glib::translate::*;
+use std::ptr;
+use libc::{c_uchar, ssize_t};
+
+//Other functions autogenerated in src\auto\css_provider.rs
+impl CssProvider {
+    pub fn load_from_data(&self, data: &str) -> Result<(), Error> {
+        unsafe {
+            let mut error = ptr::null_mut();
+            let _ = ffi::gtk_css_provider_load_from_data(self.to_glib_none().0, data.as_ptr() as *mut c_uchar,
```

Comment:
I'd prefer to stick to `mut_override` for switching pointer mutability. E.g. `mut_override(data.as_bytes().as_ptr())`.